mmv1/products/notebooks/api.yaml

```diff
@@ -281,6 +281,42 @@ objects:
               Defines whether the instance has the vTPM enabled.
               Enabled by default.
             default_value: true
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
This should be a top-level enum called `nicType`, it isn't nested as far as I can tell.